pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -375,7 +375,7 @@ func (l *LibvirtDomainManager) cancelMigration(vmi *v1.VirtualMachineInstance) e
 	return nil
 }
 
-func (l *LibvirtDomainManager) setMigrationResultHelper(failed bool, completed bool, reason string, abortStatus v1.MigrationAbortStatus) error {
```

Comment:
I believe you still need the argument here to set the timestamp

Reply:
Why? That's implicit, if you're calling this function you're setting a migration result, so it means the migration has ended.

The abortion in progress is a special case which should probably be separated from this function.

Reply:
>The abortion in progress is a special case which should probably be separated from this function.

But as of now it is not so we should not set `EndTimestamp` on every call.

Reply:
It doesn't get set if the abortion is in progress.

Reply:
We have call `l.setMigrationAbortStatus(v1.MigrationAbortFailed)` as well and I think this is wrong or not?

Reply:
this is connected to https://github.com/kubevirt/kubevirt/pull/13729#discussion_r1926873812